cmd/juju/machine/list_test.go

```diff
@@ -79,7 +79,7 @@ func (s *MachineListCommandSuite) TestMachine(c *gc.C) {
 	context, err := testing.RunCommand(c, newMachineListCommand())
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(testing.Stdout(context), gc.Equals, ""+
-		"MACHINE  STATE    DNS       INS-ID               SERIES  AZ\n"+
+		"Machine  State    DNS       Inst Id              Series  AZ\n"+
```

Comment:
This should be either "Inst ID" as with "Model UUID" columns or "Inst id" as with "Last connection"